### ISSUE TITLE
Levers breaking when re-entering an area

### ISSUE THREAD

**Issue by Detonario** [xJon/Tekkit-2#268]
### Was this bug reported before?

- [x] No, it was not.

### Please describe the issue you are experiencing

When leaving an area, and re-entering, levers may break. My IC2C machines always remain without levers when getting back to my base. It's annoying.

**Comment by xJon** [comment]
Can you please provide more details, or an example, of the issue?

**Comment by Detonario** [comment]
Here, you can check out my world. When I leave the small factory (in the East) and then go back to the large factory, the levers will pop.

**Comment by xJon** [comment]
I'd rather not load up a world if possible - are you referring just to vanilla levers? Pop off where?
Can you maybe include a screenshot?

**Comment by Detonario** [comment]
I just tested it. It's the same issue we had back then with the orientation problem of the machines. When I fly diagonally away, the levers will break. When I fly straight, they won't.

**Comment by Detonario** [comment]
yes, vanilla levers. they pop off, from anywhere i guess. as long as you place them down.

**Comment by Detonario** [comment]
https://i.imgur.com/Y60cFOY.png

if you place them like this

**Comment by xJon** [comment]
Hmm I'm actually unable to reproduce this so far. Do you play using the latest modpack version? On a server or singleplayer?

**Comment by Detonario** [comment]
singleplayer, newest Tekkit 2 version from the TechnicLauncher. try setting the render distance low

**Comment by Detonario** [comment]
1.2.4

**Comment by xJon** [comment]
Have set my render distance on 2. How far away do you have to go?

**Comment by Detonario** [comment]
probably until the chunk is not loaded anymore. you can fly away for 10 to 15 seconds. maybe it's because of something else? i don't know

**Comment by xJon** [comment]
@Detonario Can you reproduce this issue in a new world?

**Comment by Detonario** [comment]
i wasn't able to reproduce it, sorry. i tried on a Flat world and a normal one, but the levers didn't break. there must be something wrong with my survival world. probably because it contains a lot of machines and stuff?

**Comment by xJon** [comment]
Bizarre! Perhaps you can try to make small tweaks to your world, to see what affects it? Do you have any Worldspikes?

**Comment by xJon** [comment]
Closing due to inactivity, will reopen if requested!
